.gitignore

```diff
@@ -2,4 +2,5 @@ node_modules
 .DS_Store
 dist
 coverage
-.vscode
\ No newline at end of file
+.vscode
+local.env
```

Comment:
Don't put this here. Put it in a .gitignore file in the kafka-test example.

Reply:
No, leave .vscode there. Otherwise, we have to put it in every example. Remember `local.env` is just for you now while developing.